packages/bootstrap-vue-next/src/composables/useAriaInvalid.ts

```diff
@@ -1,23 +1,24 @@
-import useBooleanish from './useBooleanish'
 import type {AriaInvalid} from '../types'
-import {computed, type MaybeRefOrGetter, toRef} from 'vue'
+import {computed, type MaybeRefOrGetter, toRef, toValue} from 'vue'
 
 export default (
   ariaInvalid: MaybeRefOrGetter<AriaInvalid | undefined>,
   state: MaybeRefOrGetter<boolean | null | undefined>
-) => {
-  const resolvedAriaInvalid = useBooleanish(ariaInvalid)
-  const resolvedState = toRef(state)
+) =>
+  computed(() => {
+    const resolvedAriaInvalid = toValue(ariaInvalid)
+    const resolvedState = toRef(state)
```

Comment:
```suggestion
    const resolvedState = toValue(state)
```

Here you've done something, but I think you did it slightly be accident. You're creating a ref inside of a computed value. This is not very performant. Instead, you should use toValue here

(The previous code had these values declared outside. This is because we didn't want to init the useBooleanish composable inside computed. toRef is the same here, as we don't want to initialize refs in computeds. But we can use toValue here as its slightly more performant)